app/jobs/content_depositor_change_event_job.rb

```diff
@@ -30,18 +27,19 @@ def log_work_event(work)
   end
   alias log_file_set_event log_work_event
 
-  def work
-    @work ||= Hyrax::ChangeContentDepositorService.call(repo_object, depositor, reset)
-  end
```

Comment:
This method call has been used for over 5 years.  So it is making less sense to me now that this would be a problem.  I originally thought this was moved here when the listener was created.  Still not ruling out the various methods are chasing each other.